src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -476,6 +476,11 @@ private int CheckAndAdjustPositionForSpeculativeAttribute(int position)
         public abstract SymbolInfo GetSymbolInfo(SelectOrGroupClauseSyntax node, CancellationToken cancellationToken = default(CancellationToken));
 
         /// <summary>
+        /// Gets the semantic information associated with a local function
+        /// </summary>
+        public abstract SymbolInfo GetSymbolInfo(LocalFunctionStatementSyntax node, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
`GetSymbolInfo` is for getting a referenced symbol. You should instead be defining `GetDeclaredSymbol`.